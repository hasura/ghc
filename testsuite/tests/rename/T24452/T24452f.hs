-- Ambiguity on the export list
{-# LANGUAGE DuplicateRecordFields #-}

module T24452f (S(foo)) where
    import AmbigPatSynA
    import AmbigPatSynB
    data S 
