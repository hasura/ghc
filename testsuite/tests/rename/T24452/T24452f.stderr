T24452f.hs:4:17: [GHC-87543]
     Ambiguous occurrence ‘foo’.
      It could refer to
         either the field ‘foo’ of pattern synonym ‘AmbigPatSynA.MkT’,
                imported from ‘AmbigPatSynA’ at T24452f.hs:5:5-23
                (and originally defined at AmbigPatSynA.hs:4:15-17),
             or the field ‘foo’ of pattern synonym ‘AmbigPatSynB.MkT’,
                imported from ‘AmbigPatSynB’ at T24452f.hs:6:5-23
                (and originally defined at AmbigPatSynB.hs:4:15-17).
     In the export: S(foo)